README.md

```diff
@@ -20,7 +20,8 @@ module.exports = {
     access_token:         '...',
     access_token_secret:  '...'
   },
-  FILTERS: []
+  FILTERS: [],
+  HANDLE: '(botname, i.e. vegassist or @vegassist)'
```

Comment:
I think the parenthesis here are a bit confusing, might want to just do `HANDLE: 'botname'`